### ISSUE TITLE
A11y_Visual Studio Code Client_Extension_Clear Extension_Screen Reader: The contrast ratio of focus indicator for "Clear extension search result" button and background is (1.595:1) which is less than required 3:1.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#203649]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yWCAG2.1; #A11yTCS; #A11ySev3; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.4.11; #Non-Text Contrast;
### 
Environment and OS details:
Visual Studio Code insider version 1.86.0
Microsoft Windows 11 Enterprise 22H2 Build 22621.3007

### Repro Steps:

1. Open Visual Studio Code Insiders editor
2. Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3. Tab till "Clear extension search result" button.
5. Verify whether the contrast ratio of focus indicator for "Clear extension search result" button and background is (1.595:1) which is less than required 3:1.

### Actual:
The contrast ratio of focus indicator for "Clear extension search result" button and background is (1.595:1) which is less than required 3:1.

### Expected:
The contrast ratio of focus indicator for "Clear extension search result" button should be greater than or equal to required 3:1 So that it can be clearly visible to the user and user should be able to differentiate between the selected and not selected control.

### User Impact:
low vision users will face difficulty to identify and locate the focus and differentiate b/w the selected and not selected line if the focus indicator is not clearly visible.

### Attachment:

<img width="960" alt="Bug_29" src="https://github.com/microsoft/vscode/assets/124244478/0c025678-e146-41fe-a26c-013799167305">

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment.
Version: 1.87.0-insider (user setup)
Commit: 9db1b0571e119289a2264971bdac75793965e063
Issue is not fixed as ratio is still less then 3:1.
![Screenshot (1559)](https://github.com/microsoft/vscode/assets/124244478/5842be18-a5f5-4679-8590-49f042e27448)
